src/test/ui/inference/issue-83606.rs

```diff
@@ -5,6 +5,6 @@ fn foo<const N: usize>(_: impl std::fmt::Display) -> [usize; N] {
 }
 
 fn main() {
-    let _ = foo("foo"); //<- Do not suggest `foo::<N>("foo");`!
+    let _ = foo("foo");
```

Comment:
It can't hurt to explicitly check for the added suggestion.

Reply:
I do check for it, see issue-83606.stderr, unless you meant something else?

Reply:
I meant the inline annotations, like `//~ HELP blah`

Reply:
It doesn't actually emit that suggestion, it suggests using an explicit type on the lhs